<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowAlertRuleDialog</class>
 <widget class="QDialog" name="ShowAlertRuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报警规则查看</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pBtn_confirmDelete{
	background-color: rgba(255, 85, 127,120);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextBrowser" name="tBrow_alertRule"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,10,5,5,5">
     <item>
      <widget class="QPushButton" name="pBtn_deleteRule">
       <property name="text">
        <string>删除规则</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_refresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_format">
       <property name="text">
        <string>缩进显示JSON</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_close">
       <property name="text">
        <string>关闭页面</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,20,5">
     <item>
      <widget class="QLabel" name="label_ruleName">
       <property name="text">
        <string>规则名称</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>lineE_ruleName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineE_ruleName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_confirmDelete">
       <property name="text">
        <string>确定删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
